src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -13,10 +13,6 @@
     <div class="container-source-repos">
       <div class="container-launch-with">
         <div *ngIf="workflow?.descriptorType === WorkflowModel.DescriptorTypeEnum.WDL">
-          <div *ngIf="(hasFileImports$ | async) || (hasHttpImports$ | async)" class="import-warning">
```

Comment:
Just noticed this is targetted for 1.12.x.; should this PR be for hotfix? Or this part extracted for another PR?

Reply:
should be 1.13 for the tickets and this PR, did I miss something?

Reply:
I meant the issue dockstore/dockstore#5028, which this block addresses, has 1.12.x as the fix version.

Reply:
Was copied out for https://github.com/dockstore/dockstore-ui2/pull/1575/files